website/source/docs/schedulers.html.md

```diff
@@ -17,20 +17,29 @@ each of these schedulers.
 The `service` scheduler is designed for scheduling long lived services that
 should never go down. As such, the `service` scheduler ranks a large portion
 of the nodes that meet the job's constraints and selects the optimal node to
-place a task group on. The `service` scheduler uses a best fit scoring algorithm influenced by Google's work on
-[Borg](https://research.google.com/pubs/pub43438.html). Ranking this larger set of candidate nodes
-increases scheduling time but provides greater guarantees about the optimality
-of a job placement, which given the service workload is highly desirable.
+place a task group on. The `service` scheduler uses a best fit scoring algorithm
+influenced by Google's work on [Borg]. Ranking this larger set of candidate
+nodes increases scheduling time but provides greater guarantees about the
+optimality of a job placement, which given the service workload is highly
+desirable.
+
+Service jobs will be considered failed by Nomad upon failure or exit
+regardless of exit code and handled according to the job's [restart]
+and [reschedule] stanzas.
```

Comment:
Maybe this could be simplified a bit?

> Service jobs are intended to run until explicitly stopped by an operator. If a service task exits it is considered a failure and handled according to the job's [restart] and [reschedule] stanzas.